### PR TITLE
Laravel updates + image size improvement

### PR DESCRIPTION
**Pull request by mterrel**
## fix: Reduce image size by only copying certs for cert setup

**Background:**

* By default, Nixpacks does a `COPY . .` before each phase, which copies all of the files from the "workspace" (git repo root) into the Docker image. If there are multiple phases, this results in duplicate files present in the Docker image. This is arguably a bug.
* Although there are some semi-standard phase names (like `setup`), the actual names of phases varies depending on what providers are used and what each provider may dynamically add.
* Prior to this change, we used the phase name `setup` to add our cert. But we don't really know what other work might be done in this phase, so we don't know what files might be needed in the phase.

**Fix:**

This changes the name of our cert setup phase to `asetup` so we don't conflict with other default Nixpacks phase names. (There are a VERY few phase names in use in Nixpacks and `asetup` should be sufficiently outside their naming pattern as to be unique.) This allows us to specify `onlyIncludeFiles` for our phase, knowing for certain that only our commands will run in this phase and therefore we tell Nixpacks to only copy the files we need. That's what avoids one duplicate set of copied workspace files.

Also, we can avoid an image `RUN` layer by combining our two commands into a single entry in the `cmds` array.

## feat: Add start-php script

**Motivation:**

The default commands that are executed for a PHP app are complex and can't be easily reused by a user as their Start Command. This makes it particularly difficult to recommend doing a migration (or really anything else) in the user's Start Command.

**Fix:**

Put the complex command into a script called `start-php` in the image and put that script in `PATH`. Then we can change the default Start Command to simply `start-php` and can also direct users to use that in more complex Start Commands in the docs.

## feat: Set DB_URL for Laravel

Copies `DATABASE_URL` to `DB_URL` only for Laravel templates (that have the `laravel` tag). This is much better behavior for Laravel's default Eloquent ORM and the most common use case for a database.

### CODE REVIEW

File: startup/buildInfo.js [hunk 1]
```diff
@@ -159,6 +159,28 @@ function dockerfileBuilder(appConfig, tags) {
     };
 }
 
+// This is the same command as nixpacks generates except it uses
+// php-fpm -D instead of backgrounding via shell ("&"). This allows
+// php-fpm to start and background itself so that it is ready to
+// accept connections when nginx is started.
+const startPhpScript = `#!/usr/bin/env bash
+node /assets/scripts/prestart.mjs /assets/nginx.template.conf /nginx.conf && (php-fpm -D -y /assets/php-fpm.conf && nginx -c /nginx.conf)
```
Comment (mvachhar) [inline startup/buildInfo.js:167]: So how would a user use this to do a DB migration?

**Review by mvachhar — APPROVED** [review]
Just a question on how start-php would be used for a migration, but you can merge since it is a strict improvement regardless.

File: startup/buildInfo.js (hunk 1 above)
Comment (mterrel) [reply startup/buildInfo.js:167]: You'll see that in the docs updates. But they'd set Start Command to:

```
php artisan migrate --force && start-php
```

Comment (mvachhar) [reply startup/buildInfo.js:167]: OK, that makes sense

**Pull request merged**